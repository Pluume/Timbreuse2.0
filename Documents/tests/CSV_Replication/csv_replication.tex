\documentclass[10pt,a4paper,onecolumn]{article}
\usepackage[top=30pt,bottom=30pt,left=48pt,right=46pt]{geometry}
\usepackage{a4wide}
\usepackage{titling}
\setlength{\droptitle}{-5em}
\usepackage[version=3]{mhchem}
\usepackage{siunitx}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage[utf8]{inputenc}
\usepackage[english,francais]{babel}
\usepackage[backend=bibtex]{biblatex}
\usepackage[pdftex]{hyperref}
\setlength\parindent{0pt}
\usepackage{xcolor}
\hypersetup{colorlinks,urlcolor=blue}
\usepackage{fancyhdr}
\usepackage{lastpage}
\pagestyle{fancy}
\fancyhf{}
\chead{Francis Le Roy}
\lfoot{P1699/MCT}
\cfoot{\today}
\rfoot{Page \thepage \hspace{1pt} of \pageref{LastPage}}
% \bibliography{bib}
\renewcommand{\labelenumi}{\alph{enumi}.}



\title{Rapport de test \\ Réplication des CSV \\ sur les timbreuses connectés}

\author{Francis \textsc{Le Roy}}

\date{\today}

\begin{document}

\maketitle
\thispagestyle{fancy}

\begin{center}
\begin{tabular}{l r}
Date de réalisation : & 4 Avril, 2017 \\
Projet : & P1699 \\
\end{tabular}
\end{center}

\section{Introduction}
Lorsqu'un élève timbre sur une des timbreuses esclaves, les timbreuses sont censées s'envoyer les informations de timbrage entre elles. Le but de ce test est de vérifier si, lorsque l'on timbre sur une timbreuse, les informations sont répliquées sur l'autre timbreuse.
\section{Mise en place}
Tout d'abord, en utilisant un des lecteurs RFID (ACR1281U-C2) à disposition, il faut noter l'identifiant unique du badge que nous allons utiliser par la suite. En utilisant les deux timbreuses du centre St Roch installées le 4 Avril 2017, on timbre sur l'une des timbreuses. Puis, en se connectant en SSH sur la timbreuse sur laquelle nous n'avons \textbf{pas} timbré, nous vérifions que notre tag apparaît à l'heure à laquelle nous avons timbré.

\section{Résultats obtenus}
La distribution des timbrages fonctionne correctement car on a pu voir une réplication des informations à travers le réseau.
\section{Conclusion}
Cette fonctionnalité est donc opérationnelle.
% \printbibliography

\end{document}
