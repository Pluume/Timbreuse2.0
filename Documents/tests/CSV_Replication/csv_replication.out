\BOOKMARK [1][-]{section.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.2}{Mise en place}{}% 2
\BOOKMARK [1][-]{section.3}{R\351sultats obtenus}{}% 3
\BOOKMARK [1][-]{section.4}{Conclusion}{}% 4
